CoreScriptsRoot/Modules/Settings/Pages/ResetCharacter.lua

```diff
@@ -7,23 +7,36 @@
 
 -------------- CONSTANTS -------------
 local RESET_CHARACTER_GAME_ACTION = "ResetCharacterAction"
+local RESET_ENABLED_TEXT = "Are you sure you want to reset your character?"
+local RESET_DISABLED_TEXT = "The game doesn't allow you to reset!"
```

Comment:
RESET_DISABLED_TEXT should probably be something else. canResetChar() returning false doesn't mean that the game doesn't allow you to reset, it only means you can't currently reset.